adops/step-by-step.md

```diff
@@ -1,9 +1,241 @@
+- Set **Start time** to **Immediately**.
+- Set **End time** to **Unlimited**.
+- Set **Rate** to your [price bucket].
+- Set **Goal** type to **None**.
+
+
+![Line item delivery settings](/assets/images/ad-ops/gam-sbs/delivery-settings.png)
+
+{:start="7"}
+7. Under **Adjust delivery**, set **Rotate creatives** to **Evenly**. You can leave the defaults for everything else.
+
+
+### Targeting
+
+Under **Add targeting**, expand **Custom targeting**.
+
+{: .alert.alert-info :}
+These instructions assume you’re sending all bids to the ad server (the default). If you’re sending only the top price bid, your targeting keys will not include the bidder code. For example, rather than targeting price buckets with `hb_pb_BidderA`, you’ll target `hb_pb`. See [Send All Bids vs Top Price](/adops/send-all-vs-top-price.html) for more information.
+
+Select the price bucket key: **hb_pb_BIDDERCODE** (where BIDDERCODE is the actual code for your bidder, such as `hb_pb_BidderA`).
+
+Leave **is any of** and enter (or select) your price bucket.
+
+![Custom targeting on price bucket](/assets/images/ad-ops/gam-sbs/custom-targeting-pb.png)
+
+The following additional keys must be added for the corresponding formats:
+
+**Banner/Outstream/Native**:
+
+You can use the same line item for banner, outstream, and/or native creatives. If your ad slot could be filled by two or more of these formats, you must include the hb_format key with values specifying all expected formats. Select **hb_format_BIDDERCODE > is any of > video, banner, native**.
+
+![Custom targeting on format](/assets/images/ad-ops/gam-sbs/custom-targeting-format.png)
+
+
+{: .alert.alert-warning :}
+If you combine native with another format in a single line item, you’ll need to add creative-level targeting to designate which creatives target which format. See [Creative-level Targeting](#creative-level-targeting) below.
+
+**In-Player and Outstream Video**:
+
+Both in-player (instream) and outstream video ads target the `hb_format_BIDDERCODE=video` key-value pair, so targeting on that key alone is not enough to choose the correct line items. If you're running both instream and outstream video ads, they will most likely be separate line items, so you will need to target outstream line items to either "Inventory Type=display" or "Inventory in (list of GAM AdUnits)".
+
+
+**Long-Form (OTT) Video**:
+
+For long-form video the custom key **hb_pb_cat_dur_BIDDERCODE** is required. The value of this key breaks down like this:
+
+- *_pb* represents the price bucket. This is the currency amount entered in the **Rate** field of the **Settings** section.
+- *_cat* indicates the competitive exclusion industry code. (For engineering information, refer to the [Category Translation module](/dev-docs/modules/categoryTranslation.html)). This is the value entered in the **Label** field for the purpose of competitive exclusion. If you are not using competitive exclusion, you can omit this portion of the value.
```

Comment:
Suggest adding "Having this value in the target helps GAM choose the line items that declare the competitive exclusion label."